src/heap.h

```diff
@@ -181,6 +181,10 @@ class ObjectHeap {
     return two_space_heap_.allocate(byte_size);
   }
 
+  inline word _allocate_new_space(int byte_size) {
```

Comment:
Does this need to start with `_`? I'm okay with it for (local) consistency, but it feels like something we haven't used widely.